src/util/imagefiledata.cpp

```diff
@@ -0,0 +1,63 @@
+#include "util/imagefiledata.h"
+
+#include <QBuffer>
+#include <QByteArray>
+#include <QDebug>
+#include <QFile>
+#include <QFileInfo>
+#include <QImage>
+#include <QImageReader>
+
+#include "util/assert.h"
+
+ImageFileData::ImageFileData(
+        const QByteArray& coverArtBytes, const char* format)
+        : QImage(QImage::fromData(coverArtBytes)),
+          m_coverArtBytes(coverArtBytes) {
+    if (format != nullptr) {
+        m_coverArtFormat = QString(format);
```

Comment:
This should be a QByteArray to match the original API. There is also a += operator for QString https://doc.qt.io/qt-5/qstring.html#operator-2b-eq-7

Reply:
Thank you, I've changed it 👍